frontend/src/app/components/header-nav/header-nav.component.ts

```diff
@@ -103,8 +103,8 @@ export class HeaderNavComponent implements OnInit {
 
   public publishToPreliveButtonText(): string {
     return this.deploymentRunningForStage === 'PRELIVE'
-      ? 'Publishing to ' + this.featureToggleService.getSync('toggle.rule-deployment-prelive-label') + '...'
-      : 'Publish to ' + this.featureToggleService.getSync('toggle.rule-deployment-prelive-label') + '';
+      ? 'Publishing to ' + this.featureToggleService.getSync('toggle.deploy-prelive-fn-label') + '...'
```

Comment:
For these other toggles `FeatureToggleService` provided a service method (which then used a declared toggle name constant). Would it be worth following this approach here, too?